src/pages/BuyGMX/BuyGMX.tsx

```diff
@@ -0,0 +1,256 @@
+                </div>
+              ) : (
+                <div className="BuyGMXGLP-description">
+                  <Trans>You can transfer AVAX from other networks to Avalanche using any of the below options:</Trans>
+                </div>
+              )}
+              <div className="buttons-group">
+                {TRANSFER_EXCHANGES.filter((e) => e.networks.includes(chainId)).map((exchange) => {
+                  const icon = importImage(exchange.icon);
+                  return (
+                    <Button key={exchange.name} href={exchange.link} imgSrc={icon}>
+                      {exchange.name}
+                    </Button>
+                  );
+                })}
+              </div>
+            </div>
+          </Card>
+        </div>
+      </div>
+      <Footer />
+    </div>
+  );
+}
+
+function DecentralisedExchanges({ chainId }) {
+  const isArbitrum = chainId === ARBITRUM;
+  return (
+    <Card title={t`Buy GMX from a Decentralized Exchange`}>
+      <div className="App-card-content">
+        {isArbitrum ? (
+          <div className="exchange-info-group">
+            <div className="BuyGMXGLP-description">
+              <Trans>Buy GMX from Uniswap (make sure to select Arbitrum):</Trans>
+            </div>
+            <div className="buttons-group col-1">
+              <Button imgSrc={Uniswap} href={EXTERNAL_LINKS.buyGmx.main[chainId]}>
+                <Trans>Uniswap</Trans>
+              </Button>
+            </div>
+          </div>
+        ) : (
+          <div className="exchange-info-group">
+            <div className="BuyGMXGLP-description">
+              <Trans>Buy GMX from Traderjoe:</Trans>
+            </div>
+            <div className="buttons-group col-1">
+              <Button imgSrc={Traderjoe} href={EXTERNAL_LINKS.buyGmx.main[chainId]}>
+                <Trans>TraderJoe</Trans>
+              </Button>
+            </div>
+          </div>
+        )}
+        <div className="exchange-info-group">
+          <div className="BuyGMXGLP-description">
+            <Trans>Buy GMX using Decentralized Exchange Aggregators:</Trans>
+          </div>
+          <div className="buttons-group">
+            {DECENTRALISED_AGGRIGATORS.filter((e) => e.networks.includes(chainId)).map((exchange) => {
+              const icon = importImage(exchange.icon);
+              return (
+                <Button key={exchange.name} imgSrc={icon} href={exchange.link}>
+                  <Trans>{exchange.name}</Trans>
+                </Button>
+              );
+            })}
+          </div>
+        </div>
+        <div className="exchange-info-group">
+          <div className="BuyGMXGLP-description">
+            <Trans>Buy GMX using any token from any network:</Trans>
+          </div>
+          <div className="buttons-group">
+            <Button href={EXTERNAL_LINKS.bungee[chainId]} imgSrc={Bungee}>
```

Comment:
I thought it could be written a little easier if you change the format of the config:

```
const EXTERNAL_LINKS = {
 [ARBITRUM]: {
    bungee: ',
     banxa: '...'
}
}
```

Then you don't have to pass the chainId every time
```
const links = EXTERNAL_LINKS[chainId]
...
links.bungee
```

Reply:
Done!